apps/react/src/features/anime/components/AnimeListItem/components/Info.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC, memo } from 'react';
+import { Typography } from '@mui/material';
+
+interface AnimeInfoProps {
```

Comment:
No need to duplicate component name in interface.

```suggestion
interface Props {
```